gpu-operator/gpu-operator-confidential-containers.rst

```diff
@@ -275,6 +276,13 @@ Perform the following steps to install and verify the Confidential Containers Op
 
       $ kubectl label node <node-name> nvidia.com/gpu.workload.config=vm-passthrough
 
+   Alternatively, set the default sandbox workload to ``vm-passthrough`` to run |project-name| on all worker nodes:
```

Comment:
Question: Do you think it is also worth highlighting the `sandboxWorkloads.defaultWorkload` option in the `Install the NVIDIA GPU Operator` procedure? The user will likely be setting this option at install time, so it may be just as important, or even more important, to mention it there.

Reply:
Good sugg.  Do you mind taking a peek at the new row?

https://nvidia.github.io/cloud-native-docs/review/pr-42/gpu-operator/latest/getting-started.html#chart-customization-options

Reply:
I think the new row looks good. 

Apologies if I wasn't clear enough in my original comment, but I was suggesting we add an explicit note about this option in the `Install the NVIDIA GPU Operator` procedure on the Kata containers page: https://nvidia.github.io/cloud-native-docs/review/pr-42/gpu-operator/latest/gpu-operator-kata.html#install-the-nvidia-gpu-operator

Reply:
I'm very glad you looked at this again.  The previous commit suggested changing the cluster policy before installing the Operator--would have ended in tears.  TY again.

Reply:
I neglected this PR for a while, apologies. Looks good now!